test/modules/op/conv_transpose2d.py

```diff
@@ -27,6 +30,29 @@ def get_example_inputs(self):
         return (torch.randn(1, 16, 50, 100),)
 
 
+@skip(
+    reason="luci-interpreter does not support dynamic shape yet && onert does not support TransposeConv yet"
```

Comment:
```suggestion
    reason="luci-interpreter does not support dynamic shape yet && onert does not support ConvTranspose yet"
```
(optional) maybe..?

Reply:
'TransposeConv' is a Circle Operator name :-D